pkg/database/database.go

```diff
@@ -91,6 +99,21 @@ func (d Database) Initialize() error {
 			return err
 		}
 	}
+	if d.cfg.Database.DisableRedoLog {
```

Comment:
I have a plan to support other database product, ex: postgresql.
It should not to apply this exec if database product can not support this ALTER.
Please add Type detection and checking supported version.

Reply:
You need to add only `mysql` check. don't need to consider about other database. Because it's not appear for NOW!